test/multiverse/suites/rack/response_content_type_test.rb

```diff
@@ -35,17 +35,17 @@ def test_records_response_content_type_on_analytics_events
       assert_equal('application/xml', get_last_analytics_event[2][:'response.headers.contentType'])
     end
 
-    def test_skips_response_content_type_if_middleware_tracing_disabled
+    def test_records_response_content_type_if_middleware_tracing_disabled
       with_config(:disable_middleware_instrumentation => true) do
         rsp = get('/', {'override-content-type' => 'application/json'})
 
         assert_equal('application/json', rsp.headers['Content-Type'])
-        refute get_last_analytics_event[2][:'response.headers.contentType']
+        assert get_last_analytics_event[2][:'response.headers.contentType']
```

Comment:
Since `contentType` is also recorded with this change, should we mention that in the changelog as well?

Reply:
that's a good idea, i hadn't thought about that part when i was writing the changelog entry. d37132b24f08ffa32b060c71a129c6bd3447628a